#goal
int32 input
nav_msgs/Path nav_path
---
#result
float32 travel_dist

---
#feedback
float32 delta_angles
